chapters/BP_032_en.md

```diff
@@ -24,13 +24,10 @@
 
 ### Description
 
-Ensure that CSS and JavaScript code is not inlined in the HTML code of the page, with the possible exception of 
-configuration variables for JavaScript objects.
+Ensure that CSS and JavaScript code is not embedded in the HTML code of the page, except for configuration variables for JavaScript objects.
```

Comment:
Inline is correct, that's what front dev say.

The rest is formatting if I'm not mistaken.

Reply:
@rlemaire is right